gamma_inc_test.go

```diff
@@ -2,11 +2,13 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package mathext
+package mathext_test
```

Comment:
The standard library is a bit of an outlier though, because a lot of the standard libraries are testing themselves, and so it has to be done like this in order to avoid circular imports. Otherwise, dot imports are strongly discouraged, and there's no need here.